runtime/ops/worker_host.rs

```diff
@@ -1,32 +1,31 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
 
-use crate::permissions::Permissions;
-use crate::web_worker::run_web_worker;
-use crate::web_worker::WebWorker;
-use crate::web_worker::WebWorkerHandle;
-use crate::web_worker::WorkerEvent;
-use deno_core::error::generic_error;
-use deno_core::error::AnyError;
-use deno_core::error::JsError;
+use crate::permissions::{
+  resolve_fs_allowlist, PermissionState, Permissions, UnaryPermission,
+};
+use crate::web_worker::{
+  run_web_worker, WebWorker, WebWorkerHandle, WorkerEvent,
+};
+
+use deno_core::error::{custom_error, generic_error, AnyError, JsError};
 use deno_core::futures::channel::mpsc;
-use deno_core::serde_json;
-use deno_core::serde_json::json;
-use deno_core::serde_json::Value;
-use deno_core::BufVec;
-use deno_core::ModuleSpecifier;
-use deno_core::OpState;
-use deno_core::ZeroCopyBuf;
-use serde::Deserialize;
+use deno_core::serde::de::{self, SeqAccess};
+use deno_core::serde::{Deserialize, Deserializer};
+use deno_core::serde_json::{self, json, Value};
+use deno_core::{BufVec, ModuleSpecifier, OpState, ZeroCopyBuf};
```

Comment:
Use one import per line, otherwise git diffs are hard to read, unfortunately `rustfmt` doesn't enforce that